pages/login.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import Head from 'next/head';
+import Root from '../components/common/Root';
+import Footer from '../components/common/Footer';
+import commerce from '../lib/commerce';
+
+class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      isError: false,
+      message: null,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.loginCustomer = this.loginCustomer.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      email: event.target.value
+    });
+  }
+
+  /**
+   * Send login email
+   *
+   * @param {Event} e
+   */
+  loginCustomer(e) {
+    e.preventDefault();
+    const urlParams = new URLSearchParams(window.location.search);
+    const myParam = urlParams.get('token');
+    console.log(myParam);
+
+    const userEmail = this.state.email
+    // Reset messaging.
+    this.setState({
+      isError: false,
+      message: null
+    });
+
+    return commerce.customer.login(
+        userEmail,
+        `${window.location.origin}/login/callback?token={token}` //@TODO: verify this is correct with Auth handler.
+      ).then(() => {
+      this.setState({
+        isError: false,
+        message: [
+          'If that email address exists in our system, we\'ve just sent you a link to continue logging in!'
+        ]
+      });
+    })
+    .catch((error)=>{
+      this.setState({
+        isError: true,
+        message: error.data.error.errors.email
+      });
+    });
```

Comment:
```suggestion
    return commerce.customer.login(
      userEmail,
      `${window.location.origin}/login/callback?token={token}` //@TODO: verify this is correct with Auth handler.
    )
      .then(() => {
        this.setState({
          isError: false,
          message: [
            'If that email address exists in our system, we\'ve just sent you a link to continue logging in!'
          ],
        });
      })
      .catch((error)=>{
        this.setState({
          isError: true,
          message: error.data.error.errors.email,
        });
      });
```

Fixing indentation (you can use `yarn lint:fix` for this too)